go.sum

```diff
@@ -1,5 +1,7 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
+github.com/allisson/go-assert v0.2.0 h1:M2krVxs9wivsQe574G2tQW+ZRH8FzHQWAMM8Or2jI6M=
```

Comment:
@stepanvalyavskiy I'd recommend running `go mod tidy` before each commit (I'm using pre-commit hooks at `.git/hooks/pre-commit` to run it automatically)